docs/events_handling.md

```diff
@@ -0,0 +1,102 @@
+
+# Events handling
+
+## Subscribed events
+
+Precaution needs to subscribe to a number of events
+to satisfy different use cases of our users.
+
+The events are separated into two groups `pull requests
+events` and `check suites/runs events`.
+
+### Pull requests events
+
+Official GitHub documentation: [PullRequest event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
+
+* <strong>pull_request.opened</strong>
+
+*occurs - when a new pull request is created*
+
+*triggers - start of the Precaution scan*
+
```

Comment:
I'm still not a fan of the formatting choices here, what do you think of this?
#### `pull_request.opened`
- __occurs__: when a new pull request is created
- __triggers__: start of the Precaution scan

Reply:
I didn't know that markdown tricks.
A good idea.